src/main/resources/com/adobe/epubcheck/messages/MessageBundle.properties

```
@@ -64,6 +64,7 @@ HTM_058=HTML documents must be encoded in UTF-8, but UTF-16 was detected.
 HTM_059=Viewport "%1$s" property must not be defined more than once, but found values [%2$s].
 HTM_060a=EPUB reading systems must ignore secondary viewport meta elements in fixed-layout documents; viewport declaration "%1$s" will be ignored. 
 HTM_060b=EPUB reading systems must ignore viewport meta elements in reflowable documents; viewport declaration "%1$s" will be ignored.
+HTM_061="%1$s" is not a valid custom data attributes (it must have at least one character after the hyphen, be XML-compatible, and contain no ASCII upper alphas).
```

s/attributes/attribute/

I also wonder if "upper alphas" will be clear to general users. Might be safer to be more verbose with "uppercase letters".

```suggestion
HTM_061="%1$s" is not a valid custom data attribute (it must have at least one character after the hyphen, be XML-compatible, and not contain ASCII uppercase letters).
```